### DISCUSSION TITLE
XG Mobile power limiter

### DISCUSSION THREAD

**Discussion by Wolper321** [seerge/g-helper#694] [Ideas]
In original presets XG Mobile has different power values: Silent 80W, Performance (Balanced) 100w and Turbo (Manual) max power (165W in my case for 6850M). 
The main problem with the eGPU is that it can get quite loud and the predetermined power values are a bit off. I think it will be interesting for some users including me to find the optimal power level to the level of notice the unit consumes. Armoury Crate doesn't provide this feature although CPU power sliders are present.

**Comment by seerge** [comment]
@Wolper321 if it’s not available in armory - then most probably it’s not possible to control :)

**Reply by Wolper321** [reply]
this is sad
